Content.Server/Sound/Components/EmitSoundOnInteractUsingComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Shared.Sound.Components;
+
+namespace Content.Server.Sound.Components
+{
+    /// <summary>
+    /// Whenever this item is used upon by a specific entity prototype in the hand of a user, play a sound
+    /// </summary>
+    [RegisterComponent]
+    public sealed partial class EmitSoundOnInteractUsingComponent : BaseEmitSoundComponent
+    {
+        [DataField("UsedItemID")]
+        public string UsedItemID = "";
+    }
```

Comment:
Needs predicting + needs to use datafield styleguide (this is done by default without the string ID).

Reply:
I don't know what you are talking about in terms of predicting (though I think it's related to null values?) Also, I don't know where the style guide is. Is it on the dev wiki?

Reply:
predicting means the component+system should be in shared and the sound should by played using `_audio.PlayPredicted`

Reply:
Am I not supposed to use the function "TryEmitSound"? All other of the handlers appear to use it.